tpot/gp_deap.py

```diff
@@ -38,8 +38,37 @@
 import warnings
 import threading
 
-# Limit loops to generate a different individual by crossover/mutation
-MAX_MUT_LOOPS = 50
+def pick_two_individuals_eligible_for_crossover(population):
+    """ Pick two individuals from the population which can do crossover, that is, they share a primitive.
```

Comment:
Minor issue: Please remove the space between the """ and the start of the docstring text.